rocketbelt/components/navigation/rocketbelt.navigation-xl.js

```diff
@@ -271,6 +273,12 @@ limitations under the License.
                 }
 
                 _toggleExpandedEventHandlers.call(that);
+                
+                // IE 9 support.
+                if (navigator.userAgent.indexOf('MSIE') !== -1 || navigator.appVersion.indexOf('Trident/') > 0){
```

Comment:
Flexibility should only end up running on IE9 if it's included via a conditional comment. Will the site end up including/executing Flexibility for every browser?

Would prefer that it didn't since IE9 is such a small percentage of our user base. If we can make that happen, we should check for a specific version number here.